public/plugin/bbb/lib/bbb.lib.php

```diff
@@ -1898,16 +1884,21 @@ public function getConferenceUrl()
      *
      * @return array
      */
-    public function findMeetingByName($name)
+    public function findMeetingByName(string $name): ?array
     {
-        $meetingData = Database::select(
-            '*',
-            'plugin_bbb_meeting',
-            array('where' => array('meeting_name = ? AND status = 1 ' => $name)),
-            'first'
-        );
+        $em = Database::getManager();
+        /** @var ConferenceMeetingRepository $repo */
+        $repo = $em->getRepository(ConferenceMeeting::class);
+
+        $qb = $repo->createQueryBuilder('m')
+            ->where('m.meetingName = :name')
```

Comment:
The meetingName column does not exist anymore it is title now so it sould be changed like this :
->where('m.title = :name')